tests/tvos/view-with-example/tvos-view-with-example.test.js

```diff
@@ -1,84 +1,85 @@
-const mockCallSnapshot = []
+const mockCallSnapshot = [];
 
 const mockPromptResponses = {
-  nativeModuleName: { nativeModuleName: 'test view for tvOS' },
+  nativeModuleName: { nativeModuleName: "test view for tvOS" },
```

Comment:
Here is another example where `lint:fix` **does** clean this up on my Windows machine but **not** on my macOS machine.